### PR TITLE
#19 - Added registration Page

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #19 
Added Page object to represent versions page in Registrations service

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#61](https://github.com/artipie/nuget-adapter/issues/61) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/c48f2a7f-9be0-4181-86bc-0389787a4459); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artpie/nuget/http/metadata/PageTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+/**
+ * Tests for {@link Page}.
+ *
+ * @since 0.1
+ */
+class PageTest {
+
+    @Test
+    void shouldHaveLower() {
```
Comment (Vatavuk) [inline src/test/java/com/artpie/nuget/http/metadata/PageTest.java]: @olegmoz we can use Json Matchers https://github.com/g4s8/matchers-json and combine all tests in one.

File: src/main/java/com/artpie/nuget/http/metadata/Page.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import java.util.List;
+import javax.json.Json;
+import javax.json.JsonObject;
+
+/**
+ * Registrations page.
+ *
+ * @since 0.1
+ */
+final class Page {
```
Comment (Vatavuk) [inline src/main/java/com/artpie/nuget/http/metadata/Page.java]: @olegmoz how about we create `Page` interface and `RegistrationPage` as implementation?

File: src/test/java/com/artpie/nuget/http/metadata/PageTest.java (hunk 1 above)
Comment (olegmoz) [reply src/test/java/com/artpie/nuget/http/metadata/PageTest.java]: @Vatavuk thank you for pointing me to this library, code became much better

File: src/main/java/com/artpie/nuget/http/metadata/Page.java (hunk 2 above)
Comment (olegmoz) [reply src/main/java/com/artpie/nuget/http/metadata/Page.java]: @Vatavuk done

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @vatavuk[/z](https://www.0crat.com/u/vatavuk)[/z](https://www.0crat.com/u/vatavuk) didn't make CR comments

**Comment by 0crat** [comment]
Job was finished in 44 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**